### ISSUE TITLE
Git branch name Development doesn't match GitFlow Convention

### ISSUE THREAD

**Issue by mpellegrini** [egineering-llc/gitflow-helper-maven-plugin#31]
The branch name this plugin is using for what GitFlow calls develop is called 'DEVELOPMENT'.  Any reason why you are using DEVELOPMENT instead of DEVELOP?

https://github.com/egineering-llc/gitflow-helper-maven-plugin/blob/master/src/main/java/com/e_gineering/maven/gitflowhelper/GitBranchType.java#L12

**Comment by mpellegrini** [comment]
Have not fully tested this out yet but after looking at the code, I think the way around this is to configure the plugin with the following

```
<configuration> 
    ....
    <developmentBranchPattern>origin/develop</developmentBranchPattern>
    ....
</configuration>


Will have a go at this and see how things work.

**Comment by bvarner** [comment]
Yep that should work.

**Comment by bvarner** [comment]
Mainly because I wrote it to work with the layout employed by an opinionated client. ;-)

**Comment by mpellegrini** [comment]
Thanks for the quick reply.

**Comment by mpkorstanje** [comment]
Would you reconsider opening this issue and change the pattern to develop? It is neater if everything works out of the box and follows mavens preference for convention over configuration.

I think it would be more appropriate for your client to use the special configuration pattern.

**Comment by misterzero** [comment]
@bvarner - What is the right way to specify develop as development branch name? I used the syntax from above and merged to my develop branch, but it was still recognized by the plugin as 'OTHER' and skipped repo deployment step.

`<configuration> 
    ....
    <developmentBranchPattern>origin/develop</developmentBranchPattern>
    ....
</configuration>`

**Comment by bvarner** [comment]
The votes have it, it's pretty obvious most of you (us?) want develop to be the default on this one, and I have to agree.

In fact, I agree so much, that the default is already changed... Apparently I didn't reference this issue when I made the adjustment due to popular demand from wonderful folks like yourselves, and a sudden jolt of pragmatism from an unbeknownst origin. :smile: 

It looks like I did this back on commit 923a943 which looks like it was included in [1.6.0](https://github.com/egineering-llc/gitflow-helper-maven-plugin/releases/tag/1.6.0) 

@misterzero the current default pattern (post 1.6.0) is `(origin/)?develop`, with the origin/ being optional, due to the way local SCM manager invocations differ from jenkins-provided env.GIT_BRANCH resolution.
